src/core/services/theming/theme.palette.js

```diff
@@ -330,9 +330,8 @@ angular.module('material.core.theming.palette', [])
     '700': '#616161',
     '800': '#424242',
     '900': '#212121',
-    '1000': '#000000',
     'A100': '#ffffff',
-    'A200': '#eeeeee',
+    'A200': '#000000',
```

Comment:
grey doesn't have accent colors, we use it to host `white`, `black` and other colors from the spec